parachains/runtimes/assets/westmint/src/lib.rs

```diff
@@ -541,6 +560,40 @@ impl pallet_uniques::Config for Runtime {
 	type Locker = ();
 }
 
+parameter_types! {
+	pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
+	pub const NftsApprovalsLimit: u32 = 20;
+	pub const NftsItemAttributesApprovalsLimit: u32 = 30;
+	pub const NftsMaxTips: u32 = 10;
+	pub const NftsMaxDeadlineDuration: BlockNumber = 12 * 30 * DAYS;
+}
+
+impl pallet_nfts::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type CollectionId = u32;
+	type ItemId = u32;
+	type Currency = Balances;
+	type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
+	type ForceOrigin = AssetsForceOrigin;
+	type Locker = ();
+	type CollectionDeposit = CollectionDeposit;
+	type ItemDeposit = ItemDeposit;
+	type MetadataDepositBase = UniquesMetadataDepositBase;
+	type AttributeDepositBase = AttributeDepositBase;
+	type DepositPerByte = DepositPerByte;
+	type StringLimit = UniquesStringLimit;
+	type KeyLimit = KeyLimit;
+	type ValueLimit = ValueLimit;
+	type ApprovalsLimit = NftsApprovalsLimit;
+	type ItemAttributesApprovalsLimit = NftsItemAttributesApprovalsLimit;
+	type MaxTips = NftsMaxTips;
```

Comment:
This can be 'ConstU32<10>' rather than needing a param_type ?

Reply:
sure